environment-sample

```diff
@@ -23,3 +23,12 @@ SLACK_GENERAL_POST_KEY=slack_general_post_key
 CF_API_KEY=cf_api_key
 CF_API_EMAIL=cf_api_email
 NEWRELIC_API_KEY=newrelic_api_key
+
```

Comment:
It feels to me that these variables belong elsewhere:

(a) they're not to do with the application's environment (hence the "only required on the server")
(b) they're not about secrets (which this file is supposed to be about, really)

Feels to me like they might as well actually be literals in the gunicorn script...?

Reply:
For `GUNICORN_TIMEOUT` and `GUNICORN_LOG_LEVEL`, there are different values in staging and production.  And for these, and `GUNICORN_NUM_WORKERS`, we might want to change the values without committing and deploying.

Reply:
Ah OK!